src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,47 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Objects;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private static List<Book> books = List.of(
+            new Book("Core Java Volume I – Fundamentals", BigDecimal.valueOf(29.99)),
+            new Book("Effective Java", BigDecimal.valueOf(19.99)),
+            new Book("Java - The Complete Reference", BigDecimal.valueOf(35.45)));
```

Comment:
Avoid using hard-coded values for books in the main method. It's better to move this to a separate method or class responsible for initializing data, or even better, to use a database script.